vanilla-framework/vanilla-framework
# 1529: Fix inconsistent heading line-heights

## Done

- Fixed the heading line-heights so they match the [design spec](https://github.com/ubuntudesign/vanilla-design/blob/master/Typography/typography.md).
- Removed spacing variables from font-sizes in favour of rems

## QA
- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/headings/default/
- Check the headings have line-heights that match the [design spec](https://github.com/ubuntudesign/vanilla-design/blob/master/Typography/typography.md).

## Details

Fixes #1529
